scripts/test-incremental-multiple.sh

```diff
@@ -0,0 +1,36 @@
+test=$1
```

Comment:
What is this script? There's no explanation.

Reply:
That's a very helpful feature indeed! We should maybe start testing all incremental tests in this matter in the CI (with the first patch being the one we have and the second one empty), we currently don't test such multiruns at all automatically.

Reply:
There's actually an infinitude of testing scenarios possible for incremental tests:
* applying empty patches,
* reversing the before and after order,
* applying multiple patches in a row,
* pairwise application of those multiple patches,
* doing all the above in server mode instead,
* ...
I'm not sure how beneficial it would be to hack more and more of such interesting incremental testing scenarios into the existing already-convoluted script. They could be feature requests for GobExec (which we should start writing one day) such that we can design nice abstractions from the ground up to allow all these without piling hacks on hacks on hacks.